server/mergin/sync/schemas.py

```diff
@@ -172,7 +172,6 @@ class ProjectSchema(ma.SQLAlchemyAutoSchema):
     id = fields.UUID()
     files = fields.Nested(FileInfoSchema(), many=True)
     access = fields.Nested(ProjectAccessSchema())
-    access_requests = fields.Nested(ProjectAccessRequestSchema(), many=True)
```

Comment:
Can you also remove relationship from models so it is not fetched from db with project?

Reply:
cleaned from model removing backref for access_requests